tests/conftest.py

```diff
@@ -217,6 +227,67 @@ def DummyStellarator(tmpdir_factory):
     return DummyStellarator_out
 
 
+@pytest.fixture(scope="session")
+def DummyCoilSet(tmpdir_factory):
+    """Create and save a dummy coil set for testing."""
+    output_dir = tmpdir_factory.mktemp("result")
+    output_path_sym = output_dir.join("DummyCoilSet_sym.h5")
+    output_path_asym = output_dir.join("DummyCoilSet_asym.h5")
+
+    eq = get("precise_QH")
+    minor_radius = eq.compute("a")["a"]
+
+    # CoilSet with symmetry
+    num_coils = 3  # number of unique coils per half field period
+    grid = LinearGrid(rho=[0.0], M=0, zeta=2 * num_coils, NFP=eq.NFP * (eq.sym + 1))
+    with pytest.warns(UserWarning):  # because eq.NFP != grid.NFP
+        data_center = eq.axis.compute("x", grid=grid, basis="xyz")
+        data_normal = eq.compute("e^zeta", grid=grid)
+    centers = data_center["x"]
+    normals = rpz2xyz_vec(data_normal["e^zeta"], phi=grid.nodes[:, 2])
+    coils = []
+    for k in range(1, 2 * num_coils + 1, 2):
+        coil = FourierPlanarCoil(
+            current=1e6,
+            center=centers[k, :],
+            normal=normals[k, :],
+            r_n=[0, minor_radius + 0.5, 0],
+        )
+        coils.append(coil)
+    coilset_sym = CoilSet(coils, NFP=eq.NFP, sym=eq.sym)
+    coilset_sym.save(output_path_sym)
+
+    # equivalent CoilSet without symmetry
+    coilset_asym = CoilSet.from_symmetry(coilset_sym, NFP=eq.NFP, sym=eq.sym)
+    coilset_asym.save(output_path_asym)
+
+    DummyCoilSet_out = {
+        "output_path_sym": output_path_sym,
+        "output_path_asym": output_path_asym,
+    }
+    return DummyCoilSet_out
+
+
+@pytest.fixture(scope="session")
+def DummyMixedCoilSet(tmpdir_factory):
+    """Create and save a dummy mixed coil set for testing."""
+    output_dir = tmpdir_factory.mktemp("result")
+    output_path = output_dir.join("DummyMixedCoilSet.h5")
+
+    tf_coil = FourierPlanarCoil(center=[2, 0, 0], normal=[0, 1, 0], r_n=[1])
+    tf_coilset = CoilSet.linspaced_angular(tf_coil, n=4)
+    vf_coil = FourierRZCoil(R_n=3, Z_n=-1)
+    vf_coilset = CoilSet.linspaced_linear(
+        vf_coil, displacement=[0, 0, 2], n=3, endpoint=True
+    )
+    xyz_coil = FourierXYZCoil()
+    _ = MixedCoilSet((tf_coilset, vf_coilset, xyz_coil))
+
+    # full_coilset.save(output_path)  FIXME: MixedCoilSet save error
```

Comment:
@f0uriest there is an issue preventing `MixedCoilSet.save` from working. I could use your help with this

Reply:
What sort of issue?

Reply:
Error from running the above code, including the line that was commented out: 
```
ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (3,) + inhomogeneous part.
```

Reply:
Can you post the full traceback?

Reply:
I made an issue for it: https://github.com/PlasmaControl/DESC/issues/1013

Reply:
This will get resolved by #1016